dashboard/app/controllers/script_levels_controller.rb

```diff
@@ -39,7 +39,9 @@ def reset
     # If the user is signed in, continue normally.
     redirect_path = build_script_level_path(@script.starting_level)
 
-    if current_user
+    if Gatekeeper.allows('redirect_static_script', where: {script_name: @script.name})
+      redirect_to(DCDO.get("static_script_#{@script.name}", redirect_path))
```

Comment:
This doesn't do the session state reset for for signed out users, does that matter?

If the whole site is melting down, might we want to redirect ALL users to a static site without explicitly having to set up a path for every one.